src/js/services/user-service.js

```diff
@@ -17,7 +17,9 @@ export const EDITABLE_PROPERTIES = [
     'surname',
     'email',
     'signedUpAt',
-    'properties'
+    'properties',
+    'pendingClients',
```

Comment:
not sure that `clients` or `pendingClients` should be update-able by the client, it's really not something the user should have control on, right? Is the API allowing it anyway?